src/EnergyPlus/GroundTemperatureModeling/BaseGroundTemperatureModel.hh

```diff
@@ -70,6 +70,15 @@ enum class GroundTempObjType
     Num
 };
 
+constexpr std::array<std::string_view, static_cast<int>(GroundTempObjType::Num)> groundTempModelNamesUC = {
+    "SITE:GROUNDTEMPERATURE:UNDISTURBED:KUSUDAACHENBACH",
+    "SITE:GROUNDTEMPERATURE:UNDISTURBED:FINITEDIFFERENCE",
+    "SITE:GROUNDTEMPERATURE:BUILDINGSURFACE",
+    "SITE:GROUNDTEMPERATURE:SHALLOW",
+    "SITE:GROUNDTEMPERATURE:DEEP",
+    "SITE:GROUNDTEMPERATURE:FCFACTORMETHOD",
+    "SITE:GROUNDTEMPERATURE:UNDISTURBED:XING"};
+
```

Comment:
After making the `clear_state` change mentioned previously, my compiler started complaining about the `Array1D_string` member variable holding these. Something about "cannot be assigned because its copy assignment operator is implicitly deleted". 

So., I moved them to an array of `string_view` objects, which is the current recommendation for these things.